plugin/tagbar.vim

```diff
@@ -100,6 +100,7 @@ function! s:setup_options() abort
         \ ['jump_lazy_scroll', 0],
         \ ['left', 0],
         \ ['help_visibility', 0],
+        \ ['highlight_method', 'nearest-stl'],
```

Comment:
i wish the default is 'nearest' which kept previous default if so.

Reply:
see comment from above